src/models/user.js

```diff
@@ -1,41 +0,0 @@
-export default (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
-    email: {
-      allowNull: false,
-      type: DataTypes.STRING,
-    },
-    firstName: {
-      allowNull: true,
-      type: DataTypes.STRING,
-    },
-    lastName: {
-      allowNull: true,
-      type: DataTypes.STRING,
-    },
-    password: {
-      allowNull: true,
-      type: DataTypes.STRING,
-      defaultValue: 'password'
-    },
-    isAdmin: {
-      allowNull: false,
-      type: DataTypes.BOOLEAN,
-      defaultValue: false,
-    },
-    lastLogin: {
-      allowNull: true,
-      type: DataTypes.DATE,
-      defaultValue: sequelize.fn('NOW')
-    },
-    rememberInfo: {
-      allowNull: true,
-      type: DataTypes.BOOLEAN,
-      defaultValue: false,
-    },
-    phoneNumber: {
-      allowNull: true,
-      type: DataTypes.STRING
-    }
-  });
-  return User;
-};
```

Comment:
Please, why delete this file? I think any update or adjustment could be made directly on this file

Reply:
I did not delete it. It's showing on my local and also check it on github you will see it.